dev-libs/ustr/ustr-1.0.4-r9.ebuild

```diff
@@ -20,13 +21,19 @@ MULTILIB_WRAPPED_HEADERS=(
 	/usr/include/ustr-conf-debug.h
 )
 
+PATCHES=(
+	"${FILESDIR}/${P}-inline-check.patch"
+	"${FILESDIR}/${PN}-1.0.4-build-libs.patch"
+)
+
 src_prepare() {
-	epatch "${FILESDIR}/${P}-gcc_5-check.patch"
+	default
 	multilib_copy_sources
 }
 
 _emake() {
 	emake \
+		USE_STATIC=$(usex static-libs) \
```

Comment:
is there a reason you need static-libs?

Reply:
No idea. I did not change anything here, and haven't searched for consumers.

Reply:
I just saw that 'static-libs' was dropped by Aaron Bauman anyway.